index.html

```diff
@@ -1,6 +1,10 @@
 <!DOCTYPE html>
 <html>
 	<head>
+		<script type="text/javascript" src="//platform.linkedin.com/in.js">
```

Comment:
Any special reason to have this in the head? It would block page rendering until it loads.